lib/widgets/autocomplete.dart

```diff
@@ -140,50 +216,52 @@ class _ComposeAutocompleteState extends State<ComposeAutocomplete> with PerAccou
             Text(label),
           ])));
   }
+}
+
+class TopicAutocomplete extends AutocompleteField<TopicAutocompleteQuery, TopicAutocompleteResult, String> {
```

Comment:
nit in commit message:

    autocomplete: Implement targeted topic autocomplete
    
    Fixes: zulip#310

just `#310`, not `zulip#310`